cmd/maya-apiserver/app/server/volume_endpoint.go

```diff
@@ -5,31 +5,32 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/golang/glog"
 	"github.com/openebs/maya/types/v1"
 	"github.com/openebs/maya/volume/provisioners"
 )
 
-// VSMSpecificRequest is a http handler implementation. It deals with HTTP
-// requests w.r.t a single VSM.
+// VolumeSpecificRequest is a http handler implementation. It deals with HTTP
+// requests w.r.t a single Volume.
 //
 // TODO
 //    Should it return specific types than interface{} ?
-func (s *HTTPServer) VSMSpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+func (s *HTTPServer) VolumeSpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
 
 	fmt.Println("[DEBUG] Processing", req.Method, "request")
 
 	switch req.Method {
 	case "PUT", "POST":
-		return s.vsmAdd(resp, req)
+		return s.VolumeAdd(resp, req)
 	case "GET":
-		return s.vsmSpecificGetRequest(resp, req)
+		return s.VolumeSpecificGetRequest(resp, req)
 	default:
 		return nil, CodedError(405, ErrInvalidMethod)
 	}
 }
 
-// vsmSpecificGetRequest deals with HTTP GET request w.r.t a single VSM
-func (s *HTTPServer) vsmSpecificGetRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+// VolumeSpecificGetRequest deals with HTTP GET request w.r.t a single Volume
+func (s *HTTPServer) VolumeSpecificGetRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
```

Comment:
should this and the below functions be -- volumeSpecificGetRequest ?

Reply:
Yes it should be.